### PR TITLE
DOC: Add missing plots for discrete and continuous distributions

### PR DESCRIPTION
**Pull request by aloctavodia**
We already have this for continuous variables

### CODE REVIEW

File: pymc3/distributions/discrete.py [hunk 1]
```diff
@@ -27,6 +27,20 @@ class Binomial(Discrete):
 
     .. math:: f(x \mid n, p) = \binom{n}{x} p^x (1-p)^{n-x}
 
+    .. plot::
+        import matplotlib.pyplot as plt
+        import numpy as np
+        import scipy.stats as st
+        plt.style.use('seaborn-darkgrid')
```
Comment (junpenglao) [inline pymc3/distributions/discrete.py]: I think it is better to remove `plt.style.use('seaborn-darkgrid')` and keep the style the same as the ones in `continuous.py` (eg, using a `lambda` function and `plot_pdf`)

Comment (aloctavodia) [reply pymc3/distributions/discrete.py]: I agree we should use the same style for continuous and discrete variables, but I prefer this one.

**Comment by aloctavodia** [comment]
I also edit some of the plots to better reflect features of the distributions and complete the docstring for triangular distribution

**Comment by junpenglao** [comment]
LGTM - did you try to build the doc locally to see how it looks like?

**Comment by aloctavodia** [comment]
@junpenglao I don't know how to do that, but I could try. Could you give me some pointer?

**Comment by aloctavodia** [comment]
@junpenglao  building locally and testing

File: pymc3/distributions/discrete.py [hunk 2]
```diff
@@ -974,6 +985,7 @@ class ZeroInflatedNegativeBinomial(Discrete):
     The Zero-inflated version of the Negative Binomial (NB).
     The NB distribution describes a Poisson random variable
     whose rate parameter is gamma distributed.
+        The pmf of this distribution is
```
Comment (junpenglao) [inline pymc3/distributions/discrete.py]: wrong indent?

Comment (aloctavodia) [reply pymc3/distributions/discrete.py]: Yes.

**Comment by junpenglao** [comment]
Is this ready to merge?

**Pull request merged**